Fixed issues

logan-mo opened the pull request:
1) The first issue was the older HTTP request to the server returned an error code. After back-and-forth emails with the NASA Power team, they helped me make new HTTTP requests. Essentially, if we reduce the number of features per request, it works. So I modified the code to make two requests, each for half of the features, and then merge them both together.

2) The code would fail at the very last step because there was no folder called dataset. Now Setup.py does that for you.

logan-mo committed:
Fixed issues

1) The first issue was the the older http request to the server returned an error code. After back and Forth email with the NASA Power team, they helped me make new HTTTP requests. Essentially, if we reduce the number of features per request, it works. So I modified the code to make two requests, each for half of the features and then merge them both together.

2) A few of the Keras Experimental functions don't exist anymore. So replaced them with tf and np to list function.

3) The code would fail at the very last step because there was no folder called dataset. Now Setup.py does that for you.

markub3327 commented on config/locations_pruned.yml:
```diff
@@ -34,57 +29,22 @@ target_locations:
     location:
       - 9.332771
       - 78.388731
-  Solar power plant Koppal:
```
Why you deleted any location from `locations_pruned.yml`? All of these were used in my paper: https://www.mdpi.com/2076-3417/12/17/8852

markub3327 commented on setup.py:
```diff
@@ -0,0 +1,4 @@
+import os
+os.system("julia install_packages.jl")
+if not os.path.exists('dataset'):
```
This is a pure Julia project. Please don't combine it with Python.

Creating not existing path can be done in Jula too.

logan-mo replied:
Done.  Also, the changes to the locations_pruned.yml were not made by me, but by the code itself.

markub3327 replied:
What line of code made these changes?

markub3327 replied:
@logan-mo What line of code made these changes?

logan-mo replied:
@markub3327 No idea. I don't know Julia at all. But I do know If I run the code, the values in the locations_pruned are modified.

logan-mo replied:
![image](https://user-images.githubusercontent.com/63550599/224067940-a3db57b1-0421-48dd-998d-64be5615d4a8.png)
If I had to guess, this part that writes to the file probably does something.

markub3327 replied:
ok.

logan-mo committed:
Made the requested changes

1) Removed setup.py and added the Julia code in setup.jl

2) Restored the config files to their original state

logan-mo committed:
Update locations_pruned.yml

logan-mo commented:
Changes done.
